docs/source/migration/3.0.mdx

```diff
@@ -16,7 +16,52 @@ Although most of the library's concepts are the same, many APIs have changed to
 
 This page describes the most important changes, along with how to migrate an existing project from Apollo Android 2.x to Apollo Android 3.x.
 
-## Package name / group id / plugin id
+## The quick route 🚀
+
+Apollo Android 3.0 provides a few shortcuts and compatibility modes to ease the migration from 2.x.  To quickly arrive at a working state, follow these steps:
```

Comment:
```suggestion
Apollo Android 3 provides a few helpers and compatibility modes to ease the migration from 2.x. To quickly reach a working state, follow the steps below.
Once you have a working app, we strongly advise to read the rest of this document as the helpers will be removed in a future version of Apollo Android.
```

Reply:
Also, we'll need to upgrade that text once we remove the helper is out to say something like "use 3.0 first and then migrate to 3.1"